packages/devtools/lib/src/settings/settings_screen.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:vm_service/vm_service.dart';
+
+import '../framework/framework.dart';
+import '../globals.dart';
+import '../ui/analytics_platform.dart' as ga_platform;
+import '../ui/elements.dart';
+
+class FlagDetailsUI extends CoreElement {
+  FlagDetailsUI(Flag flag) : super('div', classes: 'flag-details-container') {
+    final CoreElement flagDescription =
+        div(c: 'flag-details-descriptions-container')
+          ..add(<CoreElement>[
+            CoreElement('h3', classes: 'flag-name', text: flag.name),
+            span(c: 'flag-description', text: flag.comment),
+          ]);
+
+    final CoreElement flagValues = div(c: 'flag-details-values-container')
+      ..layoutVertical()
+      ..add(<CoreElement>[
+        span(c: 'flag-value', text: flag.valueAsString),
+        span(c: 'flag-modified', text: 'Modified: ${flag.modified}')
```

Comment:
nit: change the display of this span so we show `modified` and `default`.

Reply:
Updated this to `flag.modified ? "modified" : "default"`, is this what you mean?